brave/brave-core
# [Brave News] Look for missing favicons in the FaviconService

Resolves https://github.com/brave/brave-browser/issues/26266

Brave News Direct sources, with favicons
![image](https://user-images.githubusercontent.com/7678024/197924644-ba8dc421-4dbc-4c1c-b345-c39edf82659d.png)


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Enable Brave News V2
2. Visit a page with a direct source (I like using https://reactjs.org/)
3. Add the direct source to Brave News
4. The source should have an icon
